python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -203,6 +203,8 @@ def simpleAttribute(self, node, name, value, sequence):
                 print(self.indent(), '</attribute>', file=self.out)
             else:
                 self.level += 1
+                if hasattr(value, 'accept'):
```

Comment:
That should catch them yes but I'm wanting to test for this __ever__ showing up first.